getting_started/README.md

```diff
@@ -10,10 +10,10 @@ If you have a specific question, the [FAQ](../FAQ.md) might have an answer, alth
 
 ## Installation
 
-- [Linux x86_64](linux_x86_64.md)
-- [MacOS Apple Silicon](macos_apple_silicon.md)
-- [MacOS x86_64](macos_x86_64.md)
-- [Windows](windows.md)
+- [🐧 Linux x86_64](linux_x86_64.md)
+- [🍏 MacOS Apple Silicon](macos_apple_silicon.md)
+- [🍏 MacOS x86_64](macos_x86_64.md)
+- [🟦 Windows](windows.md)
```

Comment:
I added some unicode symbols here to catch the attention of the scanning reader.